ultralytics/engine/exporter.py

```diff
@@ -813,19 +811,21 @@ def export_saved_model(self, prefix=colorstr("TensorFlow SavedModel:")):
             import tensorflow as tf  # noqa
         except ImportError:
             suffix = "-macos" if MACOS else "-aarch64" if ARM64 else "" if cuda else "-cpu"
-            version = "" if ARM64 else "<=2.13.1"
-            check_requirements((f"tensorflow{suffix}{version}", "keras"))
+            version = ">=2.0.0"
+            check_requirements(f"tensorflow{suffix}{version}")
             import tensorflow as tf  # noqa
         if ARM64:
             check_requirements("cmake")  # 'cmake' is needed to build onnxsim on aarch64
         check_requirements(
             (
+                "keras",
                 "onnx>=1.12.0",
-                "onnx2tf>=1.15.4,<=1.17.5",
+                "onnx2tf>1.17.5,<=1.22.3",
                 "sng4onnx>=1.0.1",
                 "onnxsim>=0.4.33",
                 "onnx_graphsurgeon>=0.3.26",
-                "tflite_support<=0.4.3" if IS_JETSON else "tflite_support",  # fix ImportError 'GLIBCXX_3.4.29'
```

Comment:
Ah, not sure why I removed, will replace!

Reply:
Oh wait, now I remember. I had to remove it because it was causing problems. I think an extremely low version was installing, i.e. 0.1 or so because of possibly a wierd difference between dash and underscore usage i.e. tflite-support vs tflite_support. 

I'll leave commented but not removed.